frozen_requirements.txt

```diff
@@ -11,10 +11,10 @@
 #    - remove the line for python-apt (TODO: handle packages that are updated by apt-get)
 #    - add this comment
 # 5. git checkout Dockerfile
-git+https://github.com/dcos/dcos-cli.git@f1fd38c9e72e1d521cf8120fe4948a789fb40cbc#egg=dcoscli&subdirectory=cli
-git+https://github.com/dcos/dcos-cli.git@f1fd38c9e72e1d521cf8120fe4948a789fb40cbc
-git+https://github.com/dcos/dcos-test-utils.git@3ebfc18ff9c5a1aa382311474a9192a68c98b0a7
-git+https://github.com/dcos/dcos-launch.git@dc1e116685fba5105f322b007549b7eb104cf441
+git+https://github.com/dcos/dcos-cli.git@f1fd38c9e72e1d521cf8120fe4948a789fb40cbc#egg=dcoscli&subdirectory=cli  # validator-hint: name=dcoscli version=SNAPSHOT
```

Comment:
I think that this file may actually be the preferred place for minor modifications. That's because trying to regenerate it from test_requirements will default to current latest version of everything, which might be prohibitively hard to get working.
I hope that enabling renovatebot will make it easy - just press the merge button on the PRs that it will be sending once they go green.

Reply:
That is a good point. Let's leave it here for the time being.